www/docs/src/components/Youtube.astro

```diff
@@ -0,0 +1,13 @@
+---
+import { YouTube } from 'astro-embed';
+---
+{/*
+    @jdtjenkins Because i know your going to complain about the center tag... the only way i could get the Youtube Embed to center was to wrap it in a center tag. I tried using the style tag but it didn't work. If you know a better way to center the video please let me know.
+
+    This is partially due to the internal CSS provided by astro-embed. As well as Starlight CSS
+
+    - @adammatthiesen
+*/}
+<center>
```

Comment:
the moment i added flex the whole video disappeared.... sooo feel free to pull a local copy of the branch and try to fix it... but i gave up after an HOUR of trying to make it work properly (DOING EXACTLY WHAT YOU JUST SAID)... and it would always float left...

Reply:
Fixed